scopes/lanes/lanes/lane.cmd.ts

```diff
@@ -384,11 +391,11 @@ export class LaneImportCmd implements Command {
   arguments = [{ name: 'lane', description: 'the remote lane name' }];
   alias = '';
   options = [
-    ['x', 'skip-dependency-installation', 'do not install packages of the imported components'],
+    ['x', 'skip-dependency-installation', 'do not install dependencies of the imported components'],
     [
       'p',
       'pattern <component-pattern>',
-      'switch only the specified component-pattern. works only when the workspace is empty',
+      'switch only the specified component-pattern. works only when the workspace is empty', // @david it's not clear what this command does
```

Comment:
Yeah this flag is mainly here for `bit lane import`, coz `bit lane import` actually runs `bit switch --get-all` behind the scenes.
The idea is to partially import a lane. In case you want only specific component(s) from a lane in this workspace.

Reply:
ok thanks, updated the description with that